modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/actions/Actions.js

```diff
@@ -28,7 +28,12 @@ const formatActions = (actions, itemData) => {
 	return actions
 		? actions.reduce((actions, action) => {
 				if (action.data?.permissionKey) {
-					if (itemData.actions?.[action.data.permissionKey]) {
+					if (
+						itemData.actions &&
+						Object.keys(itemData.actions)
+							.map((itemAction) => itemAction.toLowerCase())
+							.includes(action.data.permissionKey.toLowerCase())
```

Comment:
I believe we can simplify this check by using `.some()`. It would also result in a O(2N) to O(N) performance boost, although tiny in practice, as the array will always be small.